test/mailers/previews/kudo_mailer_preview.rb

```diff
@@ -0,0 +1,12 @@
+class KudoMailerPreview < ApplicationMailerPreview
+  # Sends a kudos notification
+  def batch_kudo_notification
+    user = create(:user)
+    work = create(:work)
+    guest_count = params[:guest_count] || 1
+    user_count = params[:user_count].to_i || 1
+    names = Array.new(user_count) { "User#{Faker::Alphanumeric.alpha(number: 8)}" }
+    hash = { "Work_#{work.id}": { guest_count:, names: } }
```

Comment:
This syntax is new to Ruby 3.1 and the rubocop annotation is a false positive. Adding a space would make a different cop complain. This cop accepts the formatting starting with [rubocop 1.24.0](https://github.com/rubocop/rubocop/releases/tag/v1.24.0), we are on 1.22.3.